pyprocessor/processing/encoder.py

```
@@ -183,104 +276,332 @@ def build_command(self, input_file, output_folder):
             if not m3u8_file.exists():
                 self.logger.error(
                     f"Failed to create master playlist for {input_file.name}"
                 )
                 return False
 
-            # Ensure we report 100% at the end
-            if progress_callback:
-                progress_callback(input_file.name, 100)
+            # Move files from temp directory to output folder
+            self.logger.info(
+                f"Moving encoded files from temporary directory to output folder"
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)